src/main/java/com/google/graphgeckos/dashboard/api/components/BuildInfo.java

```diff
@@ -0,0 +1,53 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// Represents the response json with the aggregated buildbot data.
+public class BuildInfo {
+
+  // git commit hash of the tested changes commit
+  private final String commitHash;
+  /*
+   the relative time (time since the application was started),
+   allows to sort received build info in the descending order (latest goes first)
+   */
+  private final String timestamp;
```

Comment:
It's not clear what event this timestamp field represents: the comments says "the relative time ..." but doesn't say _of what_.
You also should describe the format of this field: format, units, etc. But note my other comment about updating the field type to more suitable Java-native data type for time.